src/chapters/errors-and-debugging/categories-of-errors.rst

```diff
@@ -0,0 +1,170 @@
+the plane taking flight. 
+
+Compile-Time Errors (aka "Syntax Errors")
+------------------------------------------
+
+.. index:: syntax
+
+C# can only execute a program if the program is syntactically correct.
+**Syntax** refers to the structure of a language (spoken, programming, or
+otherwise) and the rules about that structure. For example, in English, a
+sentence must begin with a capital letter and end with appropriate punctuation.
+
+.. index::
+   single: error; syntax
+
+A **compile-time error** is a violation of the formal rules for a given language.  This can also 
+be referred to as a **syntax error**.
+
+.. admonition:: Examples
+
+   this sentence contains a syntax error. 
+
+   So does this one
+
+For most readers of English, a few syntax errors are not a significant problem.
+Our brains are often flexible enough to determine the intended meaning of a
+sentence even if it contains one or more syntax errors.
+
+Programming languages are not so forgiving. If there is a single syntax error anywhere in your program, 
+C# will display an error message and quit *immediately*. Since syntax is validated during the compilation stage, 
+syntax errors are the first we see when running a program.
+
+During the first few weeks of your programming career, you will probably spend a lot of time tracking down syntax errors. 
+However, as you gain experience, you will make fewer errors, and you will find your errors faster.
+
+.. admonition:: Try It!
+
+   Find the syntax errors in the program.
+
+   .. replit:: csharp
+      :linenos:
+      :slug: Syntax-Errors-CSharp
+
+      string day = Wednesday;
+      Console.WriteLine(day;
+
+
+.. admonition:: Question
+
+   What syntax errors did you find? What was the specific error message provided by C# in each case?
+
+Runtime Errors
+--------------
+
+.. index::
+   single: error; runtime
+
+.. index:: exception
+
+The second category consists of **runtime errors**, so called because they do not appear until you run the program. 
+These errors are also called **exceptions** because they usually indicate that something exceptional (and bad) has happened.
+
+Runtime errors occur during the execution phase of a program, so we will only encounter them *after* the syntax of our 
+program is completely correct.
+
+A common runtime error occurs when we try to use a variable that has not been created yet. 
+This can happen if you misspell the name of a variable, as the following example shows.
+
+.. admonition:: Example
+
+   .. sourcecode:: csharp
+      :linenos:
+
+      string firstName = "Jack";
+      Console.WriteLine(firstname);
+
+   **Console Output**
+
+   ::
+
+      main.cs(2,25): error CS0103: The name `firstname' does not exist in the current context
+         
+
+The syntax of our program is correct, but when the program executes, an error occurs at line 2. 
+We attempt to print the value of the variable ``firstname``, but such a variable does not exist.
```

Comment:
I think this example qualifies as a compiler error here since the program cannot process the var name firstname. For this section, since we're covering exceptions a few chapters later, what about just mentioning that we'll introduce examples of exceptions later - it may be useful for students to have first been exposed to methods. 
otherwise, you could pull in an example for the c# unit 2 book. like this: https://education.launchcode.org/csharp-web-development/chapters/exceptions/handling.html

Reply:
another idea: you can refer to these reasons why an exception might be thrown in c#.
https://education.launchcode.org/csharp-web-development/chapters/exceptions/introduction.html#when-exceptions-arise

and then you can use one of these cases for the concept check question at the end of the page in place of the misspelled variable example

Reply:
agree.  changed the focus of this section to what the complier messages say.